core/spec/lib/spree/app_configuration_spec.rb

```diff
@@ -136,6 +136,6 @@ class DummyClass; end;
   end
 
   it 'can access event adapter' do
-    expect(prefs.events.adapter).to be_an_instance_of(Spree::Event::Adapters::Default)
+    expect(prefs.events.adapter).to be(Spree::Config.events.adapter)
```

Comment:
This test is circular. `prefs` is `Spree::Config`, so this is just testing that `Spree::Config.events.adapter` is `Spree::Config.events.adapter`.

Reply:
Oh, correct:sweat_smile: I changed it to test what it's relevant from the app configuration point of view; the event configuration is accessible from the application configuration.